machine/delta/delta_ag5648/kernel/ismt.patch

```diff
@@ -0,0 +1,70 @@
+--- a/drivers/i2c/busses/i2c-ismt.c	2016-08-14 16:07:51.465407328 +0800
++++ b/drivers/i2c/busses/i2c-ismt.c	2016-09-02 11:15:53.010400955 +0800
+@@ -67,6 +67,8 @@
+ #include <linux/acpi.h>
+ #include <linux/interrupt.h>
+ 
++#include <linux/delay.h>
++
+ #include <asm-generic/io-64-nonatomic-lo-hi.h>
+ 
+ /* PCI Address Constants */
+@@ -192,6 +194,7 @@
+ module_param(bus_speed, uint, S_IRUGO);
+ MODULE_PARM_DESC(bus_speed, "Bus Speed in kHz (0 = BIOS default)");
+ 
++spinlock_t ismt_lock;
```

Comment:
Why is a spinlock needed here?  I find it difficult to believe this is required.

If this is a backport from upstream please add some reference to the upstream commit.